lib/Gym/environment.ex

```diff
@@ -25,14 +26,11 @@ defmodule Gyx.Gym.Environment do
     please use #{__MODULE__}.make(ENVIRONMENTNAME)\n")
     HelperAsync.call(python_session, :test, :register_handler, [self()])
 
-    {:ok, %__MODULE__{env: nil,
-                      current_state: nil,
-                      session: python_session,
-                      action_space: nil}}
+    {:ok, %__MODULE__{env: nil, current_state: nil, session: python_session, action_space: nil}}
```

Comment:
por lo general dijera que esta linea no cambio, entonces para que manchar la historia de git, pero si se lee mas agusto así..

Reply:
It was a `mix format` decision :c 
Should I ignore it to maintain the previous format?

Reply:
I though about mix format, but it shouldn't have changed.. you changed some config/version or something?

Reply:
The problem was I did not use format from the beginning :c